.github/workflows/pr.yml

```diff
@@ -4,6 +4,10 @@ on:
   pull_request:
     branches: [main]
 
+concurrency:
```

Comment:
This is to stop the previous build if another commit was pushed to the same branch.